packages/instant/src/components/zero_ex_instant_overlay.tsx

```diff
@@ -31,7 +31,7 @@ export const ZeroExInstantOverlay: React.StatelessComponent<ZeroExInstantOverlay
                         />
                     </Container>
                     <Container width={{ default: 'auto', sm: '100%' }} height={{ default: 'auto', sm: '100%' }}>
-                        <ZeroExInstantContainer />
+                        <ConnectedZeroExInstantContainer />
```

Comment:
Should we update all connected components / container to follow this convention?

Reply:
I feel like I've gotten some feedback on the naming here from @steveklebanoff as well so might be good to explain my reasoning. 

Typically, `connected_` is the catch-all prefix you could give to a connected component, but when possible I like to be more specific and specify what it's connected to, especially when you could feasibly have the same component connected to two different parts of the store. For example, `SelectedAssetBuyOrderProgress` is a progress bar that tracks the progress of the buy on `selecteedAsset`, but could feasibly be used to track any buy order.

Let me know if this is too confusing and you think we should just stick with `connected_`

(in `connected_account_payment_method` I mean "the account you connected to")

Reply:
I prefer just calling everything `connected_`, as names like `SelectedAssetBuyOrderProgress` feel overly verbose to me, but I understand your reasoning and am cool w/ it